README.md

```diff
@@ -69,6 +69,14 @@ Prebuilt EOSIO software packages are available for the operating systems below.
 brew tap eosio/eosio
 brew install eosio
 ```
+Note: On MacOS 10.15 (Catalina), there is a chance to face the error below which prevents successful installation of Eosio:
+```
+Reinstalling 1 broken dependent from source: eosio/eosio/eosio
```

Comment:
Not that it is a big deal, but there is a newline between the colon (`:`) and the "eosio/eosio/eosio" string.

Reply:
Done.